controllers/storagecluster/clusterclaims.go

```diff
@@ -102,15 +101,10 @@ func (obj *ocsClusterClaim) ensureCreated(r *StorageClusterReconciler, instance
 		return reconcile.Result{}, err
 	}
 
-	var isDROptimized = "false"
-	// Set isDROptmized to "false" in case of external clusters as we currently don't have to way to determine
-	// if external cluster OSDs are using bluestore-rdr
-	if !instance.Spec.ExternalStorage.Enable {
-		isDROptimized, err = creator.getIsDROptimized()
-		if err != nil {
-			r.Log.Error(err, "failed to get cephcluster status. retrying again")
-			return reconcile.Result{}, err
-		}
+	var isDROptimized = "true"
```

Comment:
@umangachapagain 
This PR is for the [EPIC](https://issues.redhat.com/browse/RHSTOR-5722)

I don't think we we would be need `IsDROptimized` in the clusterClaims anymore?  
Should we remove it completely or just keep it to `true`?  (All 4.17 clusters will be DR optimized by default)

Reply:
Remove it.

Reply:
removed it.